src/terrain/terrain.js

```diff
@@ -800,6 +801,85 @@ export class Terrain extends Elevation {
         return null;
     }
 
+    raycastBatch(points: Point[]): vec4[] {
```

Comment:
Should we replace raycast() with raycastBatch(1) defined [here](https://github.com/mapbox/mapbox-gl-js/pull/10634/files#diff-31a4a1f0767190c162e3de763fc3378d79a0f1b509d9e54a626a6fc0a5719580R757) or would this one eventually be slower for just one ray?